.buildkite/pipeline-resource-definitions/security-solution-quality-gate/kibana-serverless-security-solution-quality-gate-detection-engine.yml

```
@@ -0,0 +1,34 @@
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-kibana-serverless-security-solution-quality-gate-detection-engine
+  description: "[MKI] Executes Cypress tests for the Detection Engine team"
+spec:
+  type: buildkite-pipeline
+  owner: 'group:security-engineering-productivity'
```

Add `system: buildkite`
```suggestion
  owner: 'group:security-engineering-productivity'
  system: buildkite
```